### PR TITLE
Nathan tinkering

### PR DESCRIPTION
**Pull request by nhatch**
* Changes to error messaging (e.g. when loading caches)
* Fixed bug with computing average loss over entire epoch
* Decreased rollout length to 1 second (instead of 5 seconds) in sequence_model.py

The rollout length thing is a longer discussion, but basically if we set it to 5 seconds, then any sequences shorter than 5 seconds are completely ignored. This is a fairly important problem, because most of our backwards-mode driving is done as part of a K turn, where the time in motion is indeed less than 5 seconds. I think the long-term solution here is to support training on sequences of various lengths, rather than requiring them all to be exactly the same. But that's a larger change. In the meantime, I think using a 1-second rollout length should be sufficient to get reasonably good dynamics models, even for short backward sequences during K turns.

### CODE REVIEW

**Comment by balbok0** [comment]
Code LGTM!

As for different rollouts I can add it to a todo list. I think a function similar to [torch ConcatDataset](https://pytorch.org/docs/stable/data.html#torch.utils.data.ConcatDataset) should work.
